shell/src/main/java/tachyon/shell/TfsShell.java

```diff
@@ -138,8 +138,8 @@ public int run(String... argv) {
     try {
       command.run(args);
       return 0;
-    } catch (IOException ioe) {
-      System.out.println(ioe.getMessage());
+    } catch (IOException e) {
+      System.out.println(e.getMessage());
```

Comment:
Do we have a log file for TFSShell actions? If not where are you thinking of logging to?